packages/e2e/test/stake-pool/StakePoolProvider.test.ts

```diff
@@ -0,0 +1,220 @@
+
+        const { totalResultCount } = await provider.queryStakePools({
+          filters: { identifier: { values: [{ name: pool.metadata!.name }] } },
+          pagination
+        });
+        expect(totalResultCount).toBe(1);
+      });
+
+      it('using ticker as value', async () => {
+        const pool = pools.find(({ metadata }) => Boolean(metadata?.ticker));
+
+        if (!pool)
+          throw new Error("'some pools is found' test found only pools without ticker: this test can't be performed");
+
+        const { totalResultCount } = await provider.queryStakePools({
+          filters: { identifier: { values: [{ ticker: pool.metadata!.ticker }] } },
+          pagination
+        });
+        expect(totalResultCount).toBe(1);
+      });
+
+      it('using id OR ticker as value (same pool)', async () => {
+        const { totalResultCount } = await provider.queryStakePools({
+          filters: { identifier: { values: [{ id: pools[0].id }, { ticker: pools[0].metadata?.ticker }] } },
+          pagination
+        });
+        expect(totalResultCount).toBe(1);
+      });
+
+      it('using id OR ticker as value (distinct pools)', async () => {
+        const { totalResultCount } = await provider.queryStakePools({
+          filters: { identifier: { values: [{ id: pools[0].id }, { ticker: pools[1].metadata?.ticker }] } },
+          pagination
+        });
+        expect(totalResultCount).toBe(2);
+      });
+
+      it('using id AND ticker as value (same pool)', async () => {
+        const { totalResultCount } = await provider.queryStakePools({
+          filters: {
+            identifier: { _condition: 'and', values: [{ id: pools[0].id }, { ticker: pools[0].metadata?.ticker }] }
+          },
+          pagination
+        });
+        expect(totalResultCount).toBe(1);
+      });
+
+      it('using id AND ticker as value (distinct pools)', async () => {
+        const { totalResultCount } = await provider.queryStakePools({
+          filters: {
+            identifier: { _condition: 'and', values: [{ id: pools[0].id }, { ticker: pools[1].metadata?.ticker }] }
+          },
+          pagination
+        });
+        expect(totalResultCount).toBe(0);
+      });
+    });
+
+    describe('pledgeMet filter', () => {
+      it('without pledgeMet', async () => {
+        const { totalResultCount } = await provider.queryStakePools({
+          filters: { pledgeMet: false },
+          pagination
+        });
+        expect(totalResultCount).toBe(0);
+      });
+
+      it('without pledgeMet', async () => {
+        const { totalResultCount } = await provider.queryStakePools({
+          filters: { pledgeMet: true },
+          pagination
+        });
+        expect(totalResultCount).toBe(0);
+      });
+    });
+
+    describe('sort option', () => {
+      describe('sort by name', () => {
+        let filters: { identifier: { values: { id: Cardano.PoolId }[] } } = { identifier: { values: [] } };
```

Comment:
No need to set those if you're overwriting them in `beforeAll`. I had to stop and think why it has an empty filter :sweat_smile: 
```suggestion
        let filters: { identifier: { values: { id: Cardano.PoolId }[] } };
```

Reply:
removed (more than changed the type of the var)